frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -518,6 +523,18 @@ const CampCreationPage = (): React.ReactElement => {
   };
 
   const handleStepNavigation = (stepsToMove: number) => {
+    const invalidScheduleCard = !scheduledSessions.every(
+      (session) => session.dates.length !== 0,
+    );
+    if (
+      stepsToMove > 0 &&
+      currentPage === CampCreationPages.ScheduleSessionsPage &&
+      invalidScheduleCard
```

Comment:
Can you explain why all 3 checks are necessary? Why not but these checks into the const or vice versa?

Reply:
stepsToMove must be > 0 since we only want to deny the user from moving forward
For the other 2 checks, we only want to set the error state when we are on the ScheduleSessionsPage and when there's an invalidScheduleCard

Reply:
yea I think it'll be a lot better if we put it into the const tho, good suggestion!